queries.txt

```diff
@@ -0,0 +1,71 @@
+1. select * 
+from subjects 
+order by subject asc;
+
+2. select * 
+from subjects 
+order by location asc;
+
+3. select * 
+from books 
+where title ='Little Women';
+
+4. select * 
+from books 
+where title ilike '%python%';
+
+5. select * 
+from subjects 
+where location ilike '%main st%' 
+order by subject asc; 
+
```

Comment:
really good, you could also use - select * from subjects where location = 'Main St' order by subject;